src/consts/Stages.ts

```diff
@@ -83,6 +92,36 @@ export const STAGES: ReadonlyArray<STAGE> = [
     previous: "Selection",
     previousEventTitle:
       "Backward Stage Change: Package Approval to Candidate Selection",
+    subStages: [
+      {
+        key: "HRLPackageReview",
+        text: "HRL Package Review",
+        next: "OSFPackageReview",
+        nextEventTitle:
+          "Forward Stage Change: HRL Package Review to OSF Review",
+        previous: undefined,
+        previousEventTitle: undefined,
+      },
+      {
+        key: "OSFPackageReview",
+        text: "HRL Package Review",
```

Comment:
OSF/CSF Review?  Also, maybe just should be "OSF/CSF"?  And the one above just "HRL" not "HRL Review"?